{
  "name": "XCTAssertUnrecoverable",
  "version": "1.0.0",
  "summary": "Make it possible to test that universal error / logic failure occurs",
  "description": "This library makes it possible to test that universal error / logic failure occurs, even if you use `fatalError`, `preconditionFailure` and so on.\nThis library provides just one function `XCTAssertUnrecoverable`.\n\n```swift\nimport XCTest\nimport XCTAssertUnrecoverable\n\nclass ExampleTests: XCTestCase {\n    func testExample() {\n        XCTAssertUnrecoverable {\n            // some program that will be crash.\n        }\n    }\n}\n```",
  "homepage": "https://github.com/ukitaka/XCTAssertUnrecoverable.git",
  "license": {
    "type": "MIT",
    "file": "LICENSE"
  },
  "authors": {
    "ukitaka": "yuki.takahashi.1126@gmail.com"
  },
  "social_media_url": "",
  "swift_version": "4.1",
  "platforms": {
    "ios": "8.0",
    "osx": "10.9",
    "tvos": "9.0"
  },
  "source": {
    "git": "https://github.com/ukitaka/XCTAssertUnrecoverable.git",
    "tag": "1.0.0"
  },
  "source_files": [
    "Sources/XCTAssertUnrecoverable.swift",
    "Dependencies/Fortify/Sources/Fortify.swift"
  ],
  "frameworks": "XCTest"
}
